Packs/Cymulate/README.md

```diff
@@ -0,0 +1,25 @@
+
+### Cymulate Integrations
+Cymulate and Palo Alto Networks Cortex XSOAR integrations simplifies and automates the security validation to remediation process. Cymulate proactively validates security efficacy with threat intelligence-led attack simulations. The results of each assessment include data and guidance to remediate security gaps, these include exploited CVEs, IoCs and Sigma rules. In order to expedite and manage the remediation activity, cases are opened automatically in Cortex XSOAR with all the relevant information, and the case and progress is made visible from within the Cymulate platform. Furthermore security teams can automate purple team exercises, launch full kill chain APT scenarios, and create customized attack simulations to exercise the effectiveness of incident management processes and threat hunting capabilities. Integration with XDR/EDR, SIEM and Cortex XSOAR validates the effectiveness of each layer involved in the detection process. Attacks are correlated with the findings of these systems to simplify the validation of the events and alerts they create from within the Cymulate platform.
+
+## Use-cases
```

Comment:
```suggestion
### Use-cases
```

If you want this header to be smaller it needs to have more `#` I was not sure if this should be the same size as the `Cymulate Integrations` header, or smaller if it should be smaller use `#### Use-cases`